ios-xcframework/fastlane/Fastfile

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+default_platform :ios
+
+XCFRAMEWORK_PROJECT_ROOT_FOLDER = File.dirname(File.expand_path(__dir__))
+
+before_all do
+  # Ensure we use the latest version of the toolkit
+  check_for_toolkit_updates unless is_ci || ENV['FASTLANE_SKIP_TOOLKIT_UPDATE_CHECK']
+end
+
+lane :upload_xcframework_to_s3 do |options|
+  archive_type = 'tar.gz'
+  xcframework_archive_path = File.join(XCFRAMEWORK_PROJECT_ROOT_FOLDER, "build/xcframeworks/Gutenberg.#{archive_type}")
+
+  unless File.exist? xcframework_archive_path
+    UI.user_error! "Could not find XCFramework archive to upload at #{xcframework_archive_path}"
+  end
+
+  name = options.fetch(:name, nil)
+
+  if name.nil?
+    name = "Gutenberg-#{last_git_commit[:commit_hash]}.#{archive_type}"
+    UI.message "No name provided, will default to #{name}"
+  end
+
+  upload_to_a8c_s3(file: xcframework_archive_path, name_on_s3: name)
+  upload_podspec_to_s3(id: last_git_commit[:commit_hash])
+
+  next unless is_ci
+
+  tag = ENV.fetch('BUILDKITE_TAG', nil)
+
+  next if tag.nil? || tag.empty?
+
+  upload_to_a8c_s3(file: xcframework_archive_path, name_on_s3: "Gutenberg-#{tag}.#{archive_type}")
+end
+
+# rubocop:disable Metrics/AbcSize
+# rubocop:disable Metrics/MethodLength
+def upload_podspec_to_s3(id:)
+  ios_version = '13.0'
+  swift_version = '5.0'
+  gutenberg_package = JSON.parse(File.read(File.join(File.dirname(XCFRAMEWORK_PROJECT_ROOT_FOLDER), 'package.json')))
+  version = gutenberg_package['version']
+  # This ought to be read from package.json, but there isn't one at the moment.
+  # license = GUTENBERG_PACKAGE['license']
+  license = 'GPL'
```

Comment:
Have you considered moving the hard-coded iOS version, Swift version, and license to the ERB template? I feel like it's probably more easy and intuitive to update those fields in `podpsec.erb` than this script.

Reply:
Sounds good. I don't remember why I decided to inject them from here.

Reply:
See 9962787ce3019a319eb4949ed3372dd91bf338d9